embedchain/bots/__init__.py

```diff
@@ -1,4 +1,3 @@
-from embedchain.bots.poe import PoeBot
-from embedchain.bots.whatsapp import WhatsAppBot
-# TODO: fix discord import
-# from embedchain.bots.discord import DiscordBot
\ No newline at end of file
+from embedchain.bots.poe import PoeBot  # noqa: F401
+from embedchain.bots.whatsapp import WhatsAppBot  # noqa: F401
+from embedchain.bots.discord import DiscordBot  # noqa: F401
```

Comment:
this needs to be commented. There is an issue with the import.
If the comment is throwing lint error, please let me know here first.
@Dev-Khant

Reply:
```
from .EmbeddingFunctions import EmbeddingFunctions  # noqa: F401
from .Providers import Providers  # noqa: F401
from .VectorDatabases import VectorDatabases  # noqa: F401
from .VectorDimensions import VectorDimensions  # noqa: F401

```

I did it on purpose by looking at `embedchain\models\__init__.py`. Let me know if it needs to be commented

Reply:
The first and second line should stay.